pytition/petition/templates/petition/petition_list.html

```diff
@@ -67,6 +67,10 @@ <h4 class="card-title">{{ petition.title|striptags }}</h4>
              title="{% trans "View signatures" %}" aria-disabled="true">
               <span class="oi oi-spreadsheet"></span>
             </a>
+            <a href="{% url "transfer_petition" petition.id %}" class="btn btn-outline-primary btn-sm" data-toggle="tooltip"
+             title="{% trans "Transfer this petition to another account" %}" aria-disabled="true">
```

Comment:
Maybe reducing the title to `Transfer this petition` to make it shorter and more similar to the other titles?

Reply:
Ok!